src/components/Navigation/SpanInfo/Status/Status.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import { Status } from ".";
+
+// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction
+const meta: Meta<typeof Status> = {
+  title: "Navigation/SpanInfo/Status",
+  component: Status,
+  parameters: {
+    // More on how to position stories at: https://storybook.js.org/docs/react/configure/story-layout
+    layout: "fullscreen"
+  }
+};
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+// More on writing stories with args: https://storybook.js.org/docs/react/writing-stories/args
+export const Live: Story = {
+  args: {
+    lastSeen: new Date().toISOString(), // now
+    firstSeen: new Date(
```

Comment:
consider to move `newDate().getTime()` to separate single constant and reuse it in stories